Frameworks/Foundation/NSKVOSupport.mm

```diff
@@ -573,7 +591,7 @@ - (void)willChangeValueForKey:(NSString*)key {
     if (![self observationInfo]) {
         return;
     }
-    _dispatchWillChange(self, key, @{ NSKeyValueChangeKindKey : @(NSKeyValueChangeSetting) });
+    _dispatchWillChange(self, key, [NSMutableDictionary dictionaryWithObject:@(NSKeyValueChangeSetting) forKey:NSKeyValueChangeKindKey]);
```

Comment:
Oh no! @aballway this is what actually introduced the "information leak" I was talking about.